lib/src/includes/signed_video_common.h

```diff
@@ -127,16 +129,16 @@ signed_video_compare_versions(const char* version1, const char* version2);
 /**
  * @brief Checks the NALU if its a golden SEI
  *
- * Golden SEI is the NALU that is generated at the start of the stream. This SEI includes only
- * the TLV tags which are only needed once.
+ * Golden SEI is the NALU that is generated at the start of the stream. This SEI is self-signed,
+ * hence not bound to a stream and it includes only the TLV tags which are only needed once.
```

Comment:
grammar: ... not bound to a stream, and it only includes information needed once, such as the Public key.